cmd/tokens.go

```diff
@@ -0,0 +1,124 @@
+package cmd
+
+import (
+	"errors"
+)
+
+var errMissingToken = errors.New("missing token")
+
+func init() {
+	register(&tokensCmd{})
+}
+
+type tokensCmd struct {
+	Create  createTokensCmd  `command:"create" description:"Create a new access token"`
+	List    lsTokensCmd      `command:"ls" description:"List available access tokens"`
+	Compare compareTokensCmd `command:"compare" description:"Check if access token is valid"`
```

Comment:
Could be renamed to `check` and `checkTokensCmd`... or even `get` given that is what the underlying rest API uses?

Reply:
how about `validate`?

Reply:
nice, changed